tests/perfalert/test_perf_sheriff_bot.py

```diff
@@ -27,7 +27,7 @@
 # -> up to here
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture()
```

Comment:
Why have we changed this? The tests still pass without it.

Reply:
I played with some changes there and forgot to put it back.
I put it back again